### DISCUSSION TITLE
Scheduler Tool for starting node

### DISCUSSION THREAD

**Discussion by AndyMik90** [langgenius/dify#3425] [Suggestion]
Many productive workflows, like social media posting, article writing, checking emails, etc., require a scheduler so the workflow can start at set intervals. 

In make.com we can use different time intervals to set a scheduler for when the workflow should run.
![CleanShot 2024-04-12 at 11 53 21](https://github.com/langgenius/dify/assets/119136210/c936e547-3339-4988-a656-a265da5f15f7)
![CleanShot 2024-04-12 at 11 53 40](https://github.com/langgenius/dify/assets/119136210/9020bb30-e793-4ece-89b9-99eb49cfcbdf)

**Comment by crazywoola** [comment]
@VincePotato

**Comment by sanqianyuejia** [comment]
+1
